### DISCUSSION TITLE
武汉 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#327] [Announcements]
# 武汉 | Zi-Zi's Journey

Zi-Zi's Journey

https://gz-metal-cell.github.io/galleries/%E7%A0%94%E7%A9%B6%E7%94%9F/%E6%AD%A6%E6%B1%89/

**Comment by promefire** [comment]
😭夜景真美😭

**Reply by GZ-Metal-Cell** [reply]
😭啊😭武汉😭解pro凡心中的耶路撒冷😭他们没有好好学习就来到了保定一同受罪😭
